<ui version="4.0" >
 <class>widget_other</class>
 <widget class="QWidget" name="widget_other" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Widget_other</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <layout class="QVBoxLayout" name="verticalLayout_2" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Playlist</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="gamerulesList" />
     </item>
     <item>
      <widget class="QListWidget" name="modifiersList" />
     </item>
     <item>
      <widget class="QPushButton" name="startplaylist" >
       <property name="minimumSize" >
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>startPlaylist</string>
       </property>
       <property name="text" >
        <string>Run playlist</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <layout class="QVBoxLayout" name="verticalLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Commands</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="gl_startgame" >
       <property name="minimumSize" >
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>gl_startGame</string>
       </property>
       <property name="text" >
        <string>Start game</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>198</width>
         <height>338</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="3" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>695</width>
       <height>388</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
